src/js/views/admin/program/flow/flow_views.js

```diff
@@ -72,7 +74,12 @@ const HeaderView = View.extend({
     this.showOwner();
   },
   showPublished() {
-    const publishedComponent = new PublishedComponent({ model: this.model, isCompact: true });
+    const publishedComponent = new PublishedComponent({
+      model: this.model,
+      isCompact: true,
+      infoText: intl.admin.program.flowViews.flowStatusInfoText,
```

Comment:
I'm not 100% sure, but I assumed the infoText would only be there if publish was disabled.  We'll also need to do this same treatment to the flow sidebar

Reply:
Good call.